datascience/tables.py

```diff
@@ -2356,14 +2355,8 @@ def scatter(self, column_for_x, select=None, overlay=True, fit_line=False,
         options.update(vargs)
 
         x_data, y_labels =  self._split_column_and_labels(column_for_x)
-        if group is not None and colors is not None and group != colors:
-            warnings.warn("Do not pass both colors and group to scatter().")
-        if group is None and colors is not None:
-            # Backward compatibility
-            group = colors
-            # TODO: In a future release, warn that this is deprecated.
-            # Deprecated
-            # warnings.warn("scatter(colors=x) is deprecated. Use scatter(group=x)", FutureWarning)
+        if "colors" in vargs and vargs["colors"]:
```

Comment:
Now that I think about it, this should warn on run of the code instead of just the first run. Thoughts?

Reply:
Agreed.  Seems like warning every time might be preferable.